conans/test/functional/command/imports_test.py

```diff
@@ -283,3 +293,65 @@ def test_imports_symbolic_from_editable(self):
         self.assertEqual("hello world", self.consumer.load("bin/myfile.bin"))
         self.assertEqual("bye world", self.consumer.load("lib/myfile.lib"))
         self.assertEqual("bye moon", self.consumer.load("lib/myfile.a"))
+
+
+class SymbolicLinksImportsTest(unittest.TestCase):
+
+    @unittest.skipIf(os_info.is_windows and not is_windows_admin(),
```

Comment:
I use this actually:
```
def symlinks_supported():
    if not hasattr(os, "symlink"):
        return False
    tmpdir = temp_folder()
    try:
        save_files(tmpdir, {"a": ""})
        with chdir(tmpdir):
            os.symlink("a", "b")
            return os.path.islink("b")
    except OSError:
        return False
    finally:
        shutil.rmtree(tmpdir)
```

Reply:
Yes, much smarter then asking for administrator level.